src/components/custom/other-platforms.js

```diff
@@ -351,21 +371,15 @@ export const NavPlatform = ({ onClick, is_ppc, is_ppc_redirect }) => {
                         />
                     </>
                 </ROW>
+                <EU>{getDtraderText()}</EU>
+                <UK>{getDtraderText()}</UK>
```

Comment:
We have `UKEU` instead of declaring it twice.

Reply:
Ok noted , Thanks for the update Sean.